fr/community/index.md

```diff
@@ -53,5 +59,6 @@ Informations générales
 
 
 [ruby-central]: http://rubycentral.org/
+[ruby-discord]: https://discord.gg/UyQnKrT
```

Comment:
This invite is invalid.  If an invite link is going to be added it should be permanent.